api/provider/aws/task_list_test.go

```diff
@@ -17,7 +17,13 @@ func TestTask_populateTaskSummaries(t *testing.T) {
 			EntityID:   "t1",
 			EntityType: "task",
 			Key:        "name",
-			Value:      "tsk1",
+			Value:      "tname1",
+		},
+		{
+			EntityID:   "t1",
+			EntityType: "task",
+			Key:        "arn",
+			Value:      "arn1",
```

Comment:
I get that this is mocking the ARN in the test, but I wonder if we should be using the true form of an ARN here when we test it so that we're testing closer to the actual values that will be passed into the function

`arn:aws:ecs:<region>:<aws_account_id>:container-instance/aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee`

Reply:
given we don't do any string manipulation on the task arns, this is functionally equivalent while being easier to read